hair_salon/data/base_automation.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="break_booking_automation" model="base.automation">
+        <field name="name">Delete Main Event</field> 
+        <field name="model_id" ref="calendar.model_calendar_event"/>
+        <field name="action_server_ids" eval="[(6, 0, [ref('break_booking_server_action')])]"/>
+        <field name="trigger">on_unlink</field>
+        <field name="filter_domain">[("appointment_type_id", "!=", False),("x_break_start","!=","False"),("x_break_end","!=","False")]</field>
```

Comment:
weird that you add these fields to the filter_domain of break_booking_server_action, while removing the condition from delete_subpart_calendar_event_server_action

Reply:
The condition was useless.